config/config.go

```diff
@@ -1503,3 +1503,21 @@ func InitClient() error {
 
 	return nil
 }
+
+func ResetConfig() {
```

Comment:
Please add a comment / docstring to the function.

Reply:
on it : )